### DISCUSSION TITLE
How to let onNodeDragStop not fire when it's just a simple click selection

### DISCUSSION THREAD

**Discussion by sumengwang** [xyflow/xyflow#3114] [Q&A]
I found that if I just simply click the node, it will still trigger onNodeDragStop. This is not a behavior I want. How do I easily prevent this happening besides manually checking the original and current node position?

**Comment by bcakmakoglu** [comment]
A click will always cause a drag-stop event to be fired as it's tracked by mousedown (drag-star) and mouseup (drag-stop).
And seeing as a click is always a mousedown followed by a mouseup, you'll have to deal with that somehow, like comparing positions as you mentioned.
